src/integration/utils/runTests/constants/services.js

```diff
@@ -0,0 +1,342 @@
+    liveRadio: '/gahuza/bbc_gahuza_radio/liveradio',
+    mediaAssetPage: '/gahuza/amakuru-23257470',
+    // PGL: '/gahuza/amakuru-43894701',
+  },
+
+  gujarati: {
+    frontPage: '/gujarati',
+    articles: '/gujarati/articles/cr5el5kw591o',
+    mediaAssetPage: '/gujarati/other-news-23130286',
+    // PGL: '/gujarati/international-41345658',
+  },
+
+  hausa: {
+    frontPage: '/hausa',
+    articles: '/hausa/articles/c2nr6xqmnewo',
+    liveRadio: '/hausa/bbc_hausa_radio/liveradio',
+    mediaAssetPage: '/hausa/23269030',
+  },
+
+  hindi: {
+    frontPage: '/hindi',
+    articles: '/hindi/articles/c0469479x9xo',
+    liveRadio: '/hindi/bbc_hindi_radio/liveradio',
+    // PGL: '/hindi/india-50198153',
+  },
+
+  igbo: {
+    frontPage: '/igbo',
+    articles: '/igbo/articles/cr1lw620ygjo',
+    mediaAssetPage: '/igbo/media-23256786',
+    // PGL: '/igbo/afirika-49666505',
+    storyPage: '/igbo/afirika-23252735',
+  },
+
+  indonesia: {
+    frontPage: '/indonesia',
+    articles: '/indonesia/articles/c0q2zq8pzvzo',
+    liveRadio: '/indonesia/bbc_indonesian_radio/liveradio',
+    mediaAssetPage: '/indonesia/media-23269037',
+    // PGL: '/indonesia/indonesia-41635759',
+  },
+
+  japanese: {
+    frontPage: '/japanese',
+    articles: '/japanese/articles/cdd6p3r9g7jo',
+    mediaAssetPage: '/japanese/video-23248670',
+    // PGL: '/japanese/features-and-analysis-42786589',
+  },
+
+  korean: {
+    frontPage: '/korean',
+    articles: '/korean/articles/c3mn1lvz65xo',
+    liveRadio: '/korean/bbc_korean_radio/liveradio',
+    mediaAssetPage: '/korean/media-23248686',
+    // PGL: '/korean/features-41397333',
+  },
+
+  kyrgyz: {
+    frontPage: '/kyrgyz',
+    articles: '/kyrgyz/articles/c3xd4xg3rm9o',
+    liveRadio: '/kyrgyz/bbc_kyrgyz_radio/liveradio',
+    mediaAssetPage: '/kyrgyz/media-23257484',
+    // PGL: '/kyrgyz/world-40847556',
+  },
+
+  marathi: {
+    frontPage: '/marathi',
+    articles: '/marathi/articles/cp47g4myxz7o',
+    mediaAssetPage: '/marathi/media-23127353',
+    // PGL: '/marathi/india-42894522',
+  },
+
+  mundo: {
+    frontPage: '/mundo',
+    articles: '/mundo/articles/ce42wzqr2mko',
+    // PGL: '/mundo/deportes-36935058',
+  },
+
+  naidheachdan: {
+    articles: '/naidheachdan/sgeulachdan/c18q7nedn2ko',
+  },
+
+  nepali: {
+    frontPage: '/nepali',
+    articles: '/nepali/articles/cl90j9m3mn6o',
+    liveRadio: '/nepali/bbc_nepali_radio/liveradio',
+    mediaAssetPage: '/nepali/media-23269034',
+    // PGL: '/nepali/news-50627370',
+  },
+
+  news: {
+    articles: '/news/articles/cn7k01xp8kxo',
+  },
+
+  pashto: {
+    frontPage: '/pashto',
+    articles: '/pashto/articles/cyjmdl92z3ro',
+    liveRadio: '/pashto/bbc_pashto_radio/liveradio',
+    mediaAssetPage: '/pashto/media-23257523',
+    // PGL: '/pashto/arts-and-literature-50230813',
+  },
+
+  persian: {
+    frontPage: '/persian',
+    articles: '/persian/articles/cej3lzd5e0go',
+    liveRadio: '/persian/bbc_persian_radio/liveradio',
```

Comment:
For persian, there are 2 radio URLs- `/persian/bbc_persian_radio/liveradio` and `/persian/bbc_dari_radio/liveradio` 

Probably not for this PR, but I wonder if it might be worth allowing an array of paths, so that we could test multiple URLs - similar to the Cypress config. This would also allow us to test TC2, as well as CPS MAPs if required.

Reply:
@karinathomasbbc yeh we can easily do this. I'm in 2 minds about sharing the cypress config or just having a dedicated services config for these tests

Reply:
Happy to keep this config separate. Also happy to address multiple paths issue in a separate PR, to let us get this merged.